core/src/main/resources/jenkins/model/Jenkins/loginError.jelly

```diff
@@ -24,36 +24,17 @@ THE SOFTWARE.
 
 <!-- report a login error -->
 <?jelly escape-by-default='true'?>
-<j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:d="jelly:define" xmlns:l="/lib/layout" xmlns:t="/lib/hudson" xmlns:f="/lib/form">
+<j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler">
   <j:choose>
     <j:new var="h" className="hudson.Functions" />
     <j:when test="${app.setupWizard.isUsingSecurityToken()}">
       <j:set var="error" value="true"/>
       <st:include it="${app.setupWizard}" page="authenticate-security-token"/>
     </j:when>
     <j:when test="${app.isUseSecurity() and h.isAnonymous()}">
-      <!--
-        The only time the error message makes sense is when Jenkins is protected and the user failed to authenticate.
-        If the user accidentally comes to this page (by back button, etc), redirect away to prevent a confusion.
-      -->
       <st:statusCode value="401" />
-      <l:layout title="${%Login Error}">
-        <l:hasPermission permission="${app.READ}">
-          <st:include page="sidepanel.jelly" />
-        </l:hasPermission>
-        <l:main-panel>
-          <div style="margin: 2em; text-align:center; color:red; font-weight:bold">
-            ${%Invalid login information. Please try again.}
-            <br/>       
-            <a href="login?from=${request.session.getAttribute('from')}">${%Try again}</a>
-          </div>
-          <div align="middle">
-            <div style="margin-top:2em; color:gray; width:20em">
-              ${%If you are a system administrator and suspect this to be a configuration problem, see the server console output for more details.}
-            </div>
-          </div>
-        </l:main-panel>
-      </l:layout>
+      <j:set var="error" value="true"/>
+      <st:include it="${it}" page="login.jelly"/>
```

Comment:
I wonder how this impacts SSO type plugins. Do they show the `loginError` page?

Reply:
In very rare cases they may end up showing the loginError page however since the auth process is done outside of jenkins, this cases should be limited to a total failure in communication with the sso server, but the angry jenkins page is most likely displayed instead.